.github/workflows/release_win_freethreading.yml

```diff
@@ -57,7 +60,7 @@ jobs:
         $Env:ONNX_ML=1
         $Env:CMAKE_ARGS="-DONNX_USE_PROTOBUF_SHARED_LIBS=OFF -DONNX_USE_LITE_PROTO=ON -DONNX_WERROR=ON"
 
-        if ('${{ github.event_name }}' -eq 'schedule' -or '${{ github.event_name }}' -eq 'workflow_dispatch') {
+        if ('${{ github.event_name }}' -eq 'schedule' -or '${{ inputs.build_mode }}' -eq 'preview') {        
```

Comment:
```suggestion
        if ('${{ github.event_name }}' -eq 'schedule' -or '${{ inputs.build_mode }}' -eq 'preview') {
```

Reply:
I think comparing that it is “not release” over “is preview” is safer in case the variable is not set properly.